buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -691,6 +691,8 @@ class BeamModulePlugin implements Plugin<Project> {
       // Enable errorprone static analysis
       project.apply plugin: 'net.ltgt.errorprone'
 
+      project.configurations.errorprone { resolutionStrategy.force 'com.google.errorprone:error_prone_core:2.3.1' }
```

Comment:
In the description for the plugin (https://github.com/tbroyer/gradle-errorprone-plugin) it says to simply include errorprone as a dependency, like so:

```
dependencies {
    errorprone("com.google.errorprone:error_prone_core:$errorproneVersion")
}
```

Can we do it that way? Or are we forced to do it this way because the errorprone plugin is an older version than the one in the link?

Reply:
It needs to be after the plugin is applied

Reply:
It's because of how we apply the plugin and other plugins in our own plugin

Reply:
Ah yes, the fact that we cannot specify the dependency is because the old plugin we use doesn't work that way